packages/components/organisms/f-checkout/src/exceptions/exceptions.js

```diff
@@ -25,6 +27,7 @@ class PlaceOrderError extends Error {
         this.messageKey = 'errorMessages.genericServerError';
         this.eventToEmit = EventNames.CheckoutPlaceOrderFailure;
         this.logMessage = 'Place Order Failure';
+        this.shouldShowInDialog = true;
```

Comment:
Is this actually accurate? For duplicate orders, yes, we want a dialog, but for non-duplicate orders, or unknown failures, we don't.

Reply:
Damn. Good point